deploy/everoute-agent/agent.yaml

```diff
@@ -4,6 +4,7 @@ data:
   agentconfig.yaml: |
     datapathConfig:
       cnibr0: cnibr0
+    localGwIP: 100.64.1.1
```

Comment:
It's better not choose the 100.64.1.1, maybe another address in 100.64.

Reply:
ok, this address is free to modify in config file.